interfaces/core.go

```diff
@@ -67,7 +67,7 @@ type SlotRef struct {
 }
 
 // String returns the "snap:slot" representation of a slot reference.
-func (ref *SlotRef) String() string {
```

Comment:
Yes, with the pointer receiver the method would not be used on plain (non pointer) objects.